web-common/src/features/dashboards/time-controls/time-control-store.ts

```diff
@@ -88,70 +94,80 @@ function createTimeRangeSummary(
   );
 }
 
+export const timeControlStateSelector = ([
```

Comment:
This is not actually new code, just externalizing the selector function from `createTimeControlStore` (right below) so that I can reuse it elsewhere